app/helpers/search_bar_helper.rb

```diff
@@ -0,0 +1,16 @@
+module SearchBarHelper
+  def mount_search_bar(id)
```

Comment:
it would be great to have an option here to disable bookmarks and the `/` shortcut accesssor

e.g. on filters page or discovery rules form we don't want to have bookmarks as part of the search component

also if we have 2 or more search boxes on one page, the shortcut would be confusing and it should probably be always used only to activate the top level search

Reply:
Thanks, that's a really great idea !